### PR TITLE
Adding popup form

### PR DESCRIPTION
**Pull request by kulisic4**
**Added Form**

This commit adds the **Form** that Pops up when clicked on _Share Source_. 
I have added the -- _**async function handleSubmit**_ -- from SLAVO.io, inside the -- _**function NewFactForm**_ -- , that returns the form shown in the screenshot below:

1. **_Screenshot_**
Popup form

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -108,65 +108,106 @@ function NewFactForm({ setFacts, setShowForm }: any) {
 	async function handleSubmit(e: { preventDefault: () => void; }) {
 		// 1. Prevent browser reload
 		e.preventDefault();
-		console.log(text, source, category);
+        
+		console.log( "Submit Button", isValidHttpUrl(source) );
```
Comment (slavo3dev) [inline pages/index.tsx:112]: Need to remove the console.log from PR

File: pages/index.tsx [hunk 2]
```diff
@@ -108,65 +108,106 @@ function NewFactForm({ setFacts, setShowForm }: any) {
...
-					<option key={cat.name} value={cat.name}>
-						{cat.name.toUpperCase()}
-					</option>
-				))}
-			</select>
-			<button className='btn btn-large' disabled={isUploading}>
-        Post
-			</button>
-		</form>
+		<>
```
Comment (slavo3dev) [inline pages/index.tsx:143]: All this needs to be moved to separate component under the new name

folder structure 

src/components/modal/form-modal.ts

**Review by slavo3dev — CHANGES_REQUESTED** [review]
PR need some adjasments

**Comment by kulisic4** [comment]
I have removed console.log, and I have added src/components/modal/form-modal.ts file.

**Pull request merged**